### DISCUSSION TITLE
What is the last version of MeshCentral Agent ?

### DISCUSSION THREAD

**Discussion by wireks** [Ylianst/MeshCentral#5563] [General]
What is the last version of MeshCentral Agent ?
I have 2022-Dec-2, is it good ?

**Comment by fragtion** [comment]
I have the same question (and same version)

**Comment by Martin900t** [comment]
My Win64 shows Dec. 9, 2022. It is the latest. New clients, when available, should be built into the server upgrades. The clients will update themselves.

**Comment by wireks** [comment]
Yeah, but on https://github.com/Ylianst/MeshAgent update was a few days ago...

**Reply by si458** [reply]
thats just the repo for the agent software,
@Ylianst has to build the agent for every platform manually and this takes time!
even recently, it wasnt building correctly due to a MAC bug,
the agent version will never match the meshcentral version!
the is also already an enchancement request for us to show what version is the latest and tell you! #5500
